diff --git a/build/windows/gimp-plug-ins.rc.in b/build/windows/gimp-plug-ins.rc.in
index d6fbe8dc70..075164b27b 100644
--- a/build/windows/gimp-plug-ins.rc.in
+++ b/build/windows/gimp-plug-ins.rc.in
@@ -63,5 +63,5 @@ BEGIN
 END
 
 #include "winuser.h"
-1	ICON	QUOTE(TOP_SRCDIR) "/build/windows/plug-ins.ico"
-CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST QUOTE(TOP_SRCDIR) "/build/windows/gimp.manifest"
+1	ICON	"./build/windows/plug-ins.ico"
+CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST "./build/windows/gimp.manifest"
diff --git a/build/windows/gimp.rc.in b/build/windows/gimp.rc.in
index 846ad0445a..7d19f74705 100644
--- a/build/windows/gimp.rc.in
+++ b/build/windows/gimp.rc.in
@@ -64,6 +64,6 @@ BEGIN
 END
 
 #include "winuser.h"
-1	ICON	QUOTE(TOP_SRCDIR) "/subprojects/gimp-data/images/gimp.ico"
-2	ICON	QUOTE(TOP_SRCDIR) "/build/windows/fileicon.ico"
-CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST QUOTE(TOP_SRCDIR) "/build/windows/gimp.manifest"
+1	ICON	"./subprojects/gimp-data/images/gimp.ico"
+2	ICON	"./build/windows/fileicon.ico"
+CREATEPROCESS_MANIFEST_RESOURCE_ID RT_MANIFEST "./build/windows/gimp.manifest"
